src/HotChocolate/Core/src/Execution/Processing/ResolverTask.cs

```diff
@@ -34,6 +35,10 @@ private async ValueTask TryExecuteAndCompleteAsync(CancellationToken cancellatio
                     CompleteValue(success, cancellationToken);
                 }
             }
+            catch (ObjectDisposedException)
```

Comment:
ah this one is caused by not correctly handling the cancellation token source .... I will have a look at this.